MediaBrowser.Model/Users/UserPolicy.cs

```diff
@@ -104,6 +105,8 @@ public UserPolicy()
 
             AccessSchedules = Array.Empty<AccessSchedule>();
 
+            LoginAttemptsBeforeLockout = -1;
```

Comment:
Shouldn't this be null? I take it this is the default value when creating a new User Policy.

Reply:
From the other PR,

>Because I didn't have a better idea for what number should represent "this hasn't been set because legacy.

So it makes total sense to me that -1 is "off", and "0" is "old defaults" for compatibility, with a default of "0". I'm not sure if Null == 0 in C#, but even if so numbers are consistent and then the user can disable it with `-1` too, which is a prety common convention I've seen.